### DISCUSSION TITLE
Change Device Type

### DISCUSSION THREAD

**Discussion by theburdman** [aarons22/homebridge-bond#251] [Q&A]
Hi there - I have a fireplace in homebridge, but Siri is not smart. So she tries (and fails) to adjust the fireplace, but adjust random other lights in my house. Was hoping to try to convert the fireplace to a switch (it defaulted to a light?) But it is not clear how to do that. My config is dead simple, and no option exists within HomeKit. Does anyone know how to do that?

Thank you!

**Comment by dtpanic** [comment]
This is a bug in the plugin - specifically https://github.com/aarons22/homebridge-bond/blob/main/src/Services.ts

line 216. 

The fireplace is instantiated as a Homekit light. I've been meaning to submit a pull request with a fix for this but I haven't had time.
